web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -68,6 +72,9 @@ export const createAppConfigSchema = (platform: Platform) => {
     'keymap.newTab': shortcutSchema
       .default(defaultKeymap['newTab'])
       .describe(getShortcutDesc('open a new tab')),
+    'keymap.newTerminalTab': shortcutSchema
```

Comment:
Is this related to the search bar in any way? If not, I would move it to a separate PR.

Reply:
It is related! Before the search bar was introduced, you could open a new terminal tab by pressing Cmd + K, typing any command you want and then pressing Enter.

With the addition of the search bar, there's no way at the moment to open a local terminal tab from the search bar. So we added a new item to the More Options menu to the left of the profile selector. The shortcut is displayed there to the right of the action. This way the user can still open a new terminal tab using the keyboard only.

Since the plan is to backport the search bar to v12.3, I say we can just keep those commits in this PR.